### ISSUE TITLE
Getting this error

### ISSUE THREAD

**Issue by michealbjordan** [Tanguygab/TAB-Additions#30]
Don't know what this is, seems to have appeared when I updated the server to 1.19.4
```
[17:57:09 WARN]: java.lang.NullPointerException: Cannot invoke "java.lang.Class.getEnumConstants()" because "this.actionEnum" is null
[17:57:09 WARN]:        at TAB-Additions-1.7.1.jar//io.github.tanguygab.tabadditions.spigot.SpigotPlatform.addToChatComplete(SpigotPlatform.java:164)
[17:57:09 WARN]:        at TAB-Additions-1.7.1.jar//io.github.tanguygab.tabadditions.shared.features.chat.emojis.EmojiManager.loadAutoComplete(EmojiManager.java:134)
[17:57:09 WARN]:        at TAB-Additions-1.7.1.jar//io.github.tanguygab.tabadditions.shared.features.chat.ChatManager.onJoin(ChatManager.java:625)
[17:57:09 WARN]:        at TAB.v3.3.2.jar//me.neznamy.tab.shared.FeatureManagerImpl.onJoin(FeatureManagerImpl.java:141)
[17:57:09 WARN]:        at TAB.v3.3.2.jar//me.neznamy.tab.platforms.bukkit.BukkitEventListener.lambda$onJoin$1(BukkitEventListener.java:45)
[17:57:09 WARN]:        at TAB.v3.3.2.jar//me.neznamy.tab.shared.CpuManager.lambda$submit$1(CpuManager.java:84)
[17:57:09 WARN]:        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[17:57:09 WARN]:        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[17:57:09 WARN]:        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
[17:57:09 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[17:57:09 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[17:57:09 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833
```

**Comment by Tanguygab** [comment]
yeah I fixed it but didn't release it yet because the new version changes the entire config and I'm still waiting for TAB 4.0 to be released add some more things

**Comment by Tanguygab** [comment]
https://github.com/Tanguygab/TAB-Additions/releases/download/1.7.1/TAB-Additions-1.7.1-1.19.4.jar
fixed 1.7.1 for 1.19.4
